solr/test-framework/src/java/org/apache/solr/handler/package.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<!-- not a package-info.java, because we already defined this package in core/ -->
```

Comment:
We probably want to look at renaming some of the packages for the moved classes, since split packages are going to cause problems down the line anyway. Maybe ok to do this in a follow on issue if it ends up being too much here.

Reply:
I think this should be handled as a follow up. There are a bunch of existing split packages in `test-framework` which sadly this PR adds to, but follows the existing convention.